KIPs/kip-7-fungible_token.md

```diff
@@ -46,9 +49,46 @@ If the optional field is not marked, the function must be implemented.
 |[transferFrom](#transferfrom)| |function transferFrom(address _from, address _to, uint256 _value) public returns (bool success)|
 |[approve](#approve)| |function approve(address _spender, uint256 _value) public returns (bool success)|
 |[allowance](#allowance)| |function allowance(address _owner, address _spender) public view returns (uint256 remaining)|
-|[name](#name)|O|function name() public view returns (string)|
-|[symbol](#symbol)|O|function symbol() public view returns (string)|
-|[decimals](#decimals)|O|function decimals() public view returns (uint8)|
+
+Every [KIP-7](https://klaytn.github.io/kips/KIPs/kip-7-fungible_token) compliant contract MUST implement [KIP-13](https://klaytn.github.io/kips/KIPs/kip-interface_query_standard) `supportsInterface` with the identifier `0xXXXXXXXX` for common methods.
+```solidity
```

Comment:
Could you please check out this? If you like it, please make this similar to it.

https://github.com/klaytn/kips/blob/dc5999a20b1746ff8b7dee49824c9b2adbfbfd5a/KIPs/kip-17-non_fungible_token.md#metadata-extension

Reply:
Ok. I will modify this like it.

Reply:
I updated. PTAL. :)